solutions/devsprint-matheus-reis-3/DeliveryAppChallenge/Screens/Components/AddressListView/AddressListView.swift

```diff
@@ -23,19 +25,22 @@ class AddressListView: UIView {
         return tableView
     }()
 
-    init() {
+    init(dataSource: AddressListDataSource) {
+        self.dataSource = dataSource
```

Comment:
O ideal(e mais comum) é deixar a atribuição do dataSource para quem for instanciar a view, assim como acontece na `tableView`. Então, ao invés de passar o parâmetro no `init`, a atribuição seria feita no `AddressListViewController`